hortacloud-deployment/Dockerfile

```diff
@@ -0,0 +1,14 @@
+from node:16
+
+RUN apt-get update 
+
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+RUN unzip awscliv2.zip
+RUN ./aws/install
+RUN rm awscliv2.zip
+
+RUN npm install -g aws-cdk
+
+RUN git clone https://github.com/JaneliaSciComp/hortacloud/
+WORKDIR /hortacloud
+ADD .env .
```

Comment:
We don't want to add the .env file in the docker image. We will just copy it into the container once the container starts.